### PR TITLE
Fix label namespace in experiment.jsonnet

### PR DESCRIPTION
**Pull request by AkshitaB**
Related to https://github.com/allenai/allennlp-models/pull/238

### CODE REVIEW

**Review by ArjunSubramonian — COMMENTED** [review]
Before I approve, I'm unsure if this fixes all the "answers"-related bugs, as there are other places where "answers" shows up in .jsonnet files. I'm having trouble reproducing the bug locally with the related allennlp changes.

**Comment by AkshitaB** [comment]
@ArjunSubramonian I believe this is the only config that requires the fix, because that's the default namespace for the [VE model](https://github.com/allenai/allennlp-models/blob/main/allennlp_models/vision/models/visual_entailment.py#L52). For other configs that have `answers`, [the model](https://github.com/allenai/allennlp-models/blob/main/allennlp_models/vision/models/vilbert_vqa.py#L48) also has the namespace as `answers`.

**Review by ArjunSubramonian — APPROVED** [review]

**Pull request merged**
